hpc-cluster-setup/v3/compute-post-install.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+# Thanks to Yaroslav Bulatov for the implementaion of this script.
+# https://github.com/cybertronai/aws-network-benchmarks
+# Note: This script is tested on Alinux2 runnin on GPU instance with Tesla volta arch.
+
+sudo yum update -y
+sudo yum groupinstall "Development Tools" -y
+
+export INSTALL_ROOT=${HOME}
+export PATH="/usr/local/cuda/bin:$PATH"
+export LD_LIBRARY_PATH="/usr/local/cuda/lib64:$LD_LIBRARY_PATH"
+
+mkdir -p "$INSTALL_ROOT"/packages
+cd "$INSTALL_ROOT"/packages || exit
+
+export EFA_INSTALLER_FN=aws-efa-installer-latest.tar.gz
+echo "Installing EFA " $EFA_INSTALLER_FN
+
+wget https://s3-us-west-2.amazonaws.com/aws-efa-installer/$EFA_INSTALLER_FN
+tar -xf $EFA_INSTALLER_FN
+cd aws-efa-installer || exit
+sudo ./efa_installer.sh -y
+
+# echo "Installing CUDA"
+cd "$INSTALL_ROOT"/packages || exit
+wget https://developer.download.nvidia.com/compute/cuda/11.3.0/local_installers/cuda_11.3.0_465.19.01_linux.run
+chmod +x cuda_11.3.0_465.19.01_linux.run
+sudo ./cuda_11.3.0_465.19.01_linux.run --silent --override --toolkit --samples --no-opengl-libs
+
+echo 'Building nccl'
+cd "$INSTALL_ROOT"/packages || exit
+git clone https://github.com/NVIDIA/nccl.git || echo ignored
+cd nccl || exit
+git checkout tags/v2.11.4-1 -b v2.11.4-1
+# Choose compute capability 70 for Tesla V100 and 80 for Tesla A100
+# Refer https://en.wikipedia.org/wiki/CUDA#Supported_GPUs for different architecture 
+make -j src.build NVCC_GENCODE="-gencode=arch=compute_70,code=sm_70"
```

Comment:
Add both 70 and 80 capabilities for supporting A100s

NVCC_GENCODE="-gencode=arch=compute_70,code=sm_70 -gencode=arch=compute_80,code=sm_80"